docs: Instructions for adding project and environments (#550)
devopness/devopness: docs/projects/index.md

---
title: Projects

> intro: Learn how to use projects to group and manage and unlimited number of infrastructure resources.

Reviewer: ```suggestion
intro: Use projects to group and manage an unlimited number of infrastructure resources.
```
